cmd/launcher/systray/systray.go

```diff
@@ -0,0 +1,49 @@
+package systray
+
+import (
+	_ "embed"
```

Comment:
Is this used?

Reply:
yes it has to be imported in order to use the `go:embed` directive in
```
//go:embed kolide-mark-only-white.png
var kolideSystrayIcon []byte
```
but if you name the import you'll get an error that it's not in use

Reply:
That's not in this file? (I think it moved to icon files?)

Reply:
ohh woops, that's me not paying attention to the file the comment was in =), I'll remove it